build-system/tasks/check-links.js

```diff
@@ -167,12 +150,17 @@ function filterWhitelistedLinks(markdown) {
  * @return {Promise} Used to wait until the async link checker is done.
  */
 function runLinkChecker(markdownFile) {
+  // `-TEMPLATE` is a common suffix for files that may have interpolation
+  // tokens, possibly as part of their links. So we skip them.
```

Comment:
Sanity check: Can the interpolation tokens in the template file be surrounded by backticks? If so, this exemption will become unnecessary.

Reply:
How would this work when the tokens are part of an actual link?

```md
[${name} rules](https://github.com/ampproject/amphtml/blob/master/extensions/${name}/validator-${name}.protoascii) 
```

Reply:
What I suggested doesn't really work. https://github.com/ampproject/amphtml/pull/25226#issuecomment-545665221.